frontend-react/404.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
Look like this slipped in?

Reply:
Potentially autofixed from new project settings on save/autoformat, not an issue.